### PR TITLE
Reimplement get_list_series

### PR DESCRIPTION
**Pull request by gaker**
Fixes influxdata/influxdb-python#353

### CODE REVIEW

**Comment by gaker** [comment]
@aviau Let me know if you want anything additional on this.

**Comment by aviau** [comment]
@gaker By merging another PR, I have broken the possibility of merging this one.

Can you please rebase or just pull master into your branch?

**Comment by gaker** [comment]
@aviau Done.

File: influxdb/tests/client_test.py [hunk 1]
```diff
@@ -484,6 +484,98 @@ def test_get_list_database_fails(self):
         with _mocked_session(cli, 'get', 401):
             cli.get_list_database()
 
+<<<<<<< HEAD
```
Comment (aviau) [inline influxdb/tests/client_test.py]: oops

Comment (gaker) [inline influxdb/tests/client_test.py]: Whoops!  ヾ(⌐■_■)ノ♪

**Comment by gaker** [comment]
@aviau that's cleaned up now.  Sorry about that.

**Comment by aviau** [comment]
I don't really understand the response format.

```
[{'key': 'cpu_load_short,host=server01,region=us-west'}]
```

What is "key" here?

Shouldn't series just be a list like this? 
```
[ "cpu_load_short,host=server01,region=us-west"]
```

**Comment by gaker** [comment]
@aviau Seems reasonable to me.  I'll change it.

**Comment by sebito91** [comment]
Replaced by #806

**Pull request closed without merging**